src/RestaurantButton.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const RestaurantButton = props => {
+  return (
+    <div>
+      <ul>
+        <li>
+          Pizzas: {props.orders}{" "}
```

Comment:
This is looking good, well done!

One thing you may notice is that this isn't showing the number of pizza orders in the page, because you're not passing an `orders` prop to the `RestaurantButton` when you make it in `Restaurant.js` :)